client/stylesheets/_responsive.scss

```diff
@@ -14,13 +14,20 @@
     }
 
     .homeCardLi {
-     width: 210px;
+        width: 210px;
     }
 }
 
+@media (max-width: 1100px) {
+    .profileCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardImg` should be written in lowercase with hyphens